docs/Configuration.md

```diff
@@ -74,6 +74,8 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.gluten.sql.cacheWholeStageTransformerContext           | When true, `WholeStageTransformer` will cache the `WholeStageTransformerContext` when executing. It is used to get substrait plan node and native plan string.                                                                                                                                                                                                                                                                                                                                                                            | false                                                |
 | spark.gluten.sql.injectNativePlanStringToExplain             | When true, Gluten will inject native plan tree to Spark's explain output.                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | false                                                |
 | spark.gluten.sql.fallbackRegexpExpressions                   | When true, Gluten will fall back all regexp expressions to avoid any incompatibility risk.                                                                                                                                                                                                                                                                                                                                                                                                                                                | false                                                |
+| spark.gluten.native.additionalConfList                       | Comma-separated additional configurations to be passed to the native backend.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |                                                      |
```

Comment:
What' s the format, `k1=v1,k2=v2`?

Reply:
Only the keys. Updated in the pr description.

Reply:
Can you also describe it in the document, and update to spark.gluten.native.additionalConfNameList

Reply:
Refined the doc. Thanks!